### PR TITLE
feat: initial unspec config with eligability question (CMC-27)

### PR DESCRIPTION
**Pull request by swalker125**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CMC-27

### Change description ###

Added .gitmodules file to allow submodule init.
Added new ./bin script to import definition file from root.
Initial config with caseworker-cmc-solicitor user defined.
Eligability question added for start a case.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by swalker125** [comment]
Screenshot_2020-06-23 HMCTS Manage Cases

File: ccd-definition/AuthorisationCaseEvent.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "PERSONAL_INJURY",
+    "CaseEventID": "CREATE_CLAIM",
+    "UserRole": "caseworker-cmc-solicitor",
+    "CRUD": "CRUD"
```
Comment (pd-dudkiewicz) [inline ccd-definition/AuthorisationCaseEvent.json]: Wouldn't `CR` be enough?

File: charts/ucmc-service/Chart.yaml [hunk 2]
```diff
@@ -2,6 +2,6 @@ apiVersion: v1
 description: A Helm chart for ucmc-service App
 name: ucmc-service
 home: https://github.com/hmcts/ucmc-service
-version: 0.0.2
+version: 0.0.3
```
Comment (pd-dudkiewicz) [inline charts/ucmc-service/Chart.yaml:5]: don't forget to bump version in cnp-flux-config

File: ccd-definition/AuthorisationCaseField.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    "CaseFieldID": "caseHistory",
+    "UserRole": "caseworker-cmc-solicitor",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "PERSONAL_INJURY",
+    "CaseFieldID": "eligibilityQuestions",
+    "UserRole": "caseworker-cmc-solicitor",
+    "CRUD": "CRU"
```
Comment (pd-dudkiewicz) [inline ccd-definition/AuthorisationCaseField.json]: do we need all `CU` if it's only label?

**Review by pd-dudkiewicz — DISMISSED** [review]

File: ccd-definition/AuthorisationCaseEvent.json (hunk 1 above)
Comment (swalker125) [reply ccd-definition/AuthorisationCaseEvent.json]: yep you're probably right, a lot of this I just copied over from the existing json files! I will go through and clean these up

File: ccd-definition/AuthorisationCaseField.json (hunk 3 above)
Comment (swalker125) [reply ccd-definition/AuthorisationCaseField.json]: Can remove U, without C the label does not appear

**Review by pd-dudkiewicz — APPROVED** [review]

**Pull request merged**